src/main/java/com/dataguise/saas/dto/SummaryDetailsDTO.java

```diff
@@ -0,0 +1,215 @@
+        this.protectedData = protectedData;
+    }
+
+    public Integer getExposed() {
+        return exposed;
+    }
+
+    public void setExposed(Integer exposed) {
+        this.exposed = exposed;
+    }
+
+    public Integer getUnscanned() {
+        return unscanned;
+    }
+
+    public void setUnscanned(Integer unscanned) {
+        this.unscanned = unscanned;
+    }
+
+    public Integer getTotalCount() {
+        return totalCount;
+    }
+
+    public void setTotalCount(Integer totalCount) {
+        this.totalCount = totalCount;
+    }
+
+    public Integer getCloudCount() {
+        return cloudCount;
+    }
+
+    public void setCloudCount(Integer cloudCount) {
+        this.cloudCount = cloudCount;
+    }
+
+    public Integer getOnPremisesCount() {
+        return onPremisesCount;
+    }
+
+    public void setOnPremisesCount(Integer onPremisesCount) {
+        this.onPremisesCount = onPremisesCount;
+    }
+
+    public Integer getUnprocessed() {
+        return unprocessed;
+    }
+
+    public void setUnprocessed(Integer unprocessed) {
+        this.unprocessed = unprocessed;
+    }
+
+    public String getCleanSize() {
+        return cleanSize;
+    }
+
+    public void setCleanSize(String cleanSize) {
+        this.cleanSize = cleanSize;
+    }
+
+    public String getProtectedSize() {
+        return protectedSize;
+    }
+
+    public void setProtectedSize(String protectedSize) {
+        this.protectedSize = protectedSize;
+    }
+
+    public String getExposedSize() {
+        return exposedSize;
+    }
+
+    public void setExposedSize(String exposedSize) {
+        this.exposedSize = exposedSize;
+    }
+
+    public String getUnscannedSize() {
+        return unscannedSize;
+    }
+
+    public void setUnscannedSize(String unscannedSize) {
+        this.unscannedSize = unscannedSize;
+    }
+
+    public String getUnprocessedSize() {
+        return unprocessedSize;
+    }
+
+    public void setUnprocessedSize(String unprocessedSize) {
+        this.unprocessedSize = unprocessedSize;
+    }
+
+    public String getTotalSize() {
+        return totalSize;
+    }
+
+    public void setTotalSize(String totalSize) {
+        this.totalSize = totalSize;
+    }
+
+    public String getCleanPercent() {
+        return cleanPercent;
+    }
+
+    public void setCleanPercent(String cleanPercent) {
+        this.cleanPercent = cleanPercent;
+    }
+
+    public String getProtectedPercent() {
+        return protectedPercent;
+    }
+
+    public void setProtectedPercent(String protectedPercent) {
+        this.protectedPercent = protectedPercent;
+    }
+
+    public String getExposedPercent() {
+        return exposedPercent;
+    }
+
+    public void setExposedPercent(String exposedPercent) {
+        this.exposedPercent = exposedPercent;
+    }
+
+    public String getUnscannedPercent() {
+        return unscannedPercent;
+    }
+
+    public void setUnscannedPercent(String unscannedPercent) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SummaryDetailsDTO.java L199
    
    &#39;unscannedPercent&#39; hides a field.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a166567438>*